src/core/markovProcessGraph.js

```diff
@@ -692,9 +705,25 @@ export class MarkovProcessGraph {
   }
 
   toJSON(): MarkovProcessGraphJSON {
+    const nodes = Array.from(this._nodes.values());
+    const sortedNodes = sortBy(nodes, (n) => n.address);
+    const nodeIndex: Map<
+      NodeAddressT,
+      number /* index into nodeOrder */
+    > = new Map();
+    sortedNodes.forEach((n, i) => {
+      nodeIndex.set(n.address, i);
+    });
```

Comment:
Cool. We may at some point want this to also just be the internal
representation (i.e., instance variables). But we can do that whenever.